amadeus/reference_data/_location.py

```diff
@@ -17,7 +17,6 @@ def get(self, **params):
         :rtype: amadeus.Response
         :raises amadeus.ResponseError: if the request could not be completed
         '''
-        return self.client.get(
-            '/v1/reference-data/locations/{0}'.format(self.location_id),
-            **params
-        )
+        return self.client.get('/v1/reference-data/locations/{0}'
+                               .format(self.location_id), **params)
+
```

Comment:
warning: amadeus/reference_data/_location.py:22:1: W391 blank line at end of file